corehq/util/timezones/utils.py

```diff
@@ -60,3 +60,9 @@ def get_timezone_for_user(couch_user_or_id, domain):
                 return coerce_timezone_value(domain_membership.timezone)
 
     return get_timezone_for_domain(domain)
+
+
+
+def get_timezone_aware_date_for_request(date):
```

Comment:
Actually not positive this'll work since this happens in a celery task and
it doesn't have a request object. We can just use the domain though

On Sunday, September 4, 2016, dimagimon <notifications@github.com> wrote:

> In corehq/util/timezones/utils.py
> <https://github.com/dimagi/commcare-hq/pull/13100#discussion_r77458819>:
>
> > @@ -60,3 +60,9 @@ def get_timezone_for_user(couch_user_or_id, domain):
> >                  return coerce_timezone_value(domain_membership.timezone)
> >
> >      return get_timezone_for_domain(domain)
> > +
> > +
> > +
> > +def get_timezone_aware_date_for_request(date):
>
> E303 too many blank lines (3)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/13100/files/cf4dfc52f44378c667dd3bc44980e256fd585737#r77458819>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA4D8lvK2YsTab712vUyAMoXrhDIVneXks5qmzUNgaJpZM4J0oT8>
> .
>